src/apps/loyalty/server/route_handlers.tsx

```diff
@@ -60,31 +60,44 @@ export function Inquiries(req: Request, res: Response, next: NextFunction) {
   if (info.confirmed_buyer_at) {
     markCollectorAsLoyaltyApplicant(CURRENT_USER.accessToken)
       .then(profile => {
-        return res.redirect(req.baseUrl + "/thank-you")
+        res.redirect(req.baseUrl + "/thank-you")
       })
-      .catch(err => console.error(err))
-  }
-
-  IsomorphicRelay.prepareData({
-    Container: InquiriesContainer,
-    queryConfig: new CurrentUserRoute(),
-  }, res.locals.networkLayer).then(
-    ({ data, props }) => {
-      const html = renderToString(
-        <Artsy.ContextProvider currentUser={CURRENT_USER}>
-          <IsomorphicRelay.Renderer {...props} />
-        </Artsy.ContextProvider>,
-      )
-      const styles = styleSheet.rules().map(rule => rule.cssText).join("\n")
-      res.locals.sharify.data.RELAY_DATA = data
-      return res.send(renderPage({
-        styles,
-        html,
-        entrypoint: req.baseUrl + "/bundles/inquiries.js",
-        sharify: res.locals.sharify,
-        baseURL: req.baseUrl,
-      }))
+      .catch(err => {
+        if (process.env.NODE_ENV === "development") {
```

Comment:
In development, a promise being rejected will lead to the error being printed to the console, as well as sent/rendered as text. I'm guessing that might be what happens when you call `next()` and have no explicit handler?

I think that's sufficient for development, as the dev will clearly see the error message.